### PR TITLE
ADD an ignore for an eslint error about a missing dependency (puppeteer)

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: https://github.com/storybookjs/storybook/pull/6993#discussion_r298812737

## What I did
ADD a ignore rule

### CODE REVIEW

**Review by shilman — CHANGES_REQUESTED** [review]
Add this in the file instead of globally?

**Comment by ndelangen** [comment]
@shilman no, the issue is: sometimes the package is installed, sometimes it isn't.

In the case the package is installed, eslint will complain about an eslint ignore that's unnecessary. It's pretty meta, but your suggestion won't work.

**Comment by Hypnosphi** [comment]
I think it's better to add `.eslintrc.js` to storyshots-puppeteer and specify puppeteer in `coreDependencies`. That way, it won't ignore other import errors

**Review by shilman — APPROVED** [review]
LGTM tho there are some deepscan errors with the unused vars

**Comment by ndelangen** [comment]
I'm fine with those

File: addons/storyshots/storyshots-puppeteer/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const error = 2;
+const warn = 1;
+const ignore = 0;
+
+module.exports = {
+  rules: {
+    'import/no-unresolved': ignore,
```
Comment (Hypnosphi) [inline addons/storyshots/storyshots-puppeteer/.eslintrc.js]:
```
'settings': {
  'import/core-modules': ['puppeteer']
}
```

https://github.com/benmosher/eslint-plugin-import#importcore-modules

Comment (ndelangen) [reply addons/storyshots/storyshots-puppeteer/.eslintrc.js]: thanks, that works!

**Pull request merged**